app/javascript/gobierto_data/webapp/components/sets/RecentQueries.vue

```diff
@@ -18,25 +21,36 @@ import axios from 'axios'
 import { baseUrl } from "./../../../lib/commons.js";
 export default {
   name: "RecentQueries",
-  filters: {
-    truncate: function (text, length, suffix) {
-        return text.substring(0, length) + suffix;
-    },
-    replace: function(text) {
-      return text.replace(/%20/g, ' ').replace(/%/g, ' ');
+  props: {
+    tableName: {
+      type: String,
+      required: true
     }
   },
   data() {
     return {
-      items: []
+      items: [],
+      filterItemsByDataset: [],
```

Comment:
You're setting all variables you need inside `data`, but you don't render them. Remove them from here.

It's ok you deal with `this.` to scope the variables, but as a good practice, you should get used to scope with `this.` only those vars you need throughout multiple functions. Otherwise, you should use `let` and `const`

I saw this behaviour multiple times, in different PRs. Do not abuse of `this.`, but declaring vars inside `data()` is definitely wrong.